pkg/util/ipvs/ipvs_linux.go

```diff
@@ -52,9 +52,8 @@ func New(exec utilexec.Interface) Interface {
 }
 
 // EnsureVirtualServerAddressBind is part of Interface.
-func (runner *runner) EnsureVirtualServerAddressBind(vs *VirtualServer, dummyDev string) (exist bool, err error) {
-	addr := vs.Address.String() + "/32"
-	args := []string{"addr", "add", addr, "dev", dummyDev}
+func (runner *runner) EnsureVirtualServerAddressBind(addr string, dummyDev string) (exist bool, err error) {
```

Comment:
If you change the parameter from *VirtualServer to string, should it still exist in util. It seems nothing to do with IPVS now, it's more close to proxier.

Reply:
Yes, you are right, I will move it to `pkg/proxy/util/utils.go`

Reply:
Actually, I don't think `pkg/proxy/util/utils.go` is a good place since other proxiers maybe never use it. What about `pkg/proxy/ipvs/proxier.go`?

Reply:
Yeah, `pkg/proxy/ipvs/proxier.go` might be a better place.

Reply:
Put stuff in utils when you have a reason to put it there.